From: Lennart Poettering <lennart@poettering.net>
Date: Wed, 8 Apr 2020 16:12:00 +0200
Subject: units: pull in systemd-pstore.service from sysinit.target

sysinit.target is the target our early boot services are generally
pulled in from, make systemd-pstore.service not an exception of that.

Effectively this doesn't mean much, either way our unit is part of the
initial transaction.

(cherry picked from commit 167241912f51fbc0d7d0869b9af34c15b5ecc4b6)
---
 units/systemd-pstore.service.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/units/systemd-pstore.service.in b/units/systemd-pstore.service.in
index 89f34af..37fcf87 100644
--- a/units/systemd-pstore.service.in
+++ b/units/systemd-pstore.service.in
@@ -15,6 +15,7 @@ ConditionVirtualization=!container
 DefaultDependencies=no
 Wants=systemd-remount-fs.service
 After=systemd-remount-fs.service
+Before=sysinit.target
 
 [Service]
 Type=oneshot
@@ -23,4 +24,4 @@ RemainAfterExit=yes
 StateDirectory=systemd/pstore
 
 [Install]
-WantedBy=systemd-remount-fs.service
+WantedBy=sysinit.target
